web/app/model/Attribute.js

```diff
@@ -27,5 +27,11 @@ Ext.define('Traccar.model.Attribute', {
     }, {
         name: 'value',
         type: 'string'
+    }, {
```

Comment:
The reasons is:
- Determine what row is selected: position field ( speed, altitude etc.) or attribute. `aliasEditButton` is enabled only for attributes. But I just thought that `priority` can be used for such check. OK.
- I consider that we should make it comfort and load selected row to `AttributeAliasDialog` on edit click. To load we need:
  - Know, if it is already alias, and load existed record from store to form. But we have to know original lower-camel-case attribute to lookup in store.
  - If it is not alias we have to fill `attribute` form field by original lower-camel-case attribute.

I can move alias applying from renderer co controller, then `alias` field can be removed from model.

Am I missing something?